fhevm/tee_interpreter.go

```diff
@@ -72,6 +72,62 @@ func doOperationGeneric(
 	return resultHash[:], nil
 }
 
+// doEqNeoperationsGeneric is a generic function to do TEE bit shift operations
+func doEqNeOperationsGeneric(
+	environment EVMEnvironment,
+	caller common.Address,
+	input []byte,
+	runSpan trace.Span,
+	operator func(a, b *big.Int) uint64,
+	op string) ([]byte, error) {
```

Comment:
I think this format is more idiomatic in Go

```suggestion
	op string,
) ([]byte, error) {
```